esteid/smartid/tests/conftest.py

```diff
@@ -0,0 +1,53 @@
+import base64
+import uuid
+
+import pytest
+
+from esteid.tests.conftest import *  # noqa: F401, F403  -- force pytest to use fixtures from that file
+
+from ...constants import HASH_SHA512
+from ...util import generate_hash
+from ..base import SmartIDService
+from ..constants import CERTIFICATE_LEVEL_QUALIFIED, END_RESULT_OK, STATE_COMPLETE
+from ..i18n import TranslatedSmartIDService
+from ..types import AuthenticateResult
+from ..utils import get_verification_code
+
+
+@pytest.fixture(scope="function")
```

Comment:
function scope is default value, why explicitly state it?

Reply:
The magic of copypasting legacy code =)